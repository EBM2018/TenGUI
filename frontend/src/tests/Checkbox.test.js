import React from 'react';
import renderer from 'react-test-renderer';
import CheckBox from '../react/Widgets/Activity/CheckBox.js';


test('CheckBox renders correctly', () => {
  const component = renderer
    .create(<CheckBox id={1} reponse="test" fctChecked={() => 'lel'} check response="42" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
