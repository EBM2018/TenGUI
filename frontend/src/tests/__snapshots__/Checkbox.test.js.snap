// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox renders correctly 1`] = `
Array [
  <input
    onChange={[Function]}
    type="checkbox"
  />,
  <label>
     
    test
     
  </label>,
]
`;
